tidy3d/components/geometry/base.py

```diff
@@ -885,7 +885,7 @@ def from_gds(
         dilation: float = 0.0,
         sidewall_angle: float = 0,
         reference_plane: PlanePosition = "middle",
-    ) -> GeometryGroup:
+    ) -> Geometry:
```

Comment:
should be `Union[Geometry, GeometryGroup]`? (and in docstring)

Reply:
I don't think so… `GeometryGroup` is a `Geometry` type ([see here](https://github.com/flexcompute/tidy3d/blob/develop/tidy3d/components/geometry/utils.py#L13-L22)).